### PR TITLE
added data test ids

### PR DESCRIPTION
**Pull request by atlldwp**
## Links to Tickets or other pull requests
[BC-3078](https://ticketsystem.dbildungscloud.de/browse/BC-3078)

## Changes
- Assigned data-testids on different web elements

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: views/administration/school.hbs [hunk 1]
```diff
@@ -380,6 +380,7 @@
 										type="checkbox"
 										name="rocketChat"
 										value="true"
+                                        data-testid="school-admin-activate-chat-checkbox"
```
Comment (MartinSchuhmacher) [inline views/administration/school.hbs:383]: as we are already in the context of administration and its not only for activating it I would either name it `chat-checkbox` or if you would like to keep admin stuff, then `school-administration-chat-checkbox` or `administration-chat-checkbox`

File: views/administration/school.hbs [hunk 2]
```diff
@@ -407,7 +408,7 @@
 						{{#ifConfig "FEATURE_VIDEOCONFERENCE_ENABLED" true}}
 						<div class="form-group">
 							<label>
-								<input type="checkbox" name="videoconference" value="true"
+								<input type="checkbox" name="videoconference" value="true" data-testid="school-admin-video-conference-checkbox"
```
Comment (MartinSchuhmacher) [inline views/administration/school.hbs:411]: see above, either: `video-conference-checkbox` or `school-administration-video-conference-checkbox` or simply `administration-video-conference-checkbox`

File: views/administration/school.hbs [hunk 3]
```diff
@@ -426,7 +427,7 @@
 								</option>
 								{{/each}} </select> </div> {{/if}}
 						{{> "lib/components/csrfInput"}}
-						<button type="submit" class="btn btn-primary btn-submit">{{$t "administration.school.button.saveGeneralSettings" }}</button>
+						<button type="submit" class="btn btn-primary btn-submit" data-testid="school-admin-save-general-setting">{{$t "administration.school.button.saveGeneralSettings" }}</button>
```
Comment (MartinSchuhmacher) [inline views/administration/school.hbs:430]: see above

File: views/administration/school.hbs (hunk 1 above)
Comment (atlldwp) [reply views/administration/school.hbs]: Thanks for the feedback.
So I updated the name to -->  school-administration-chat-checkbox

File: views/administration/school.hbs (hunk 2 above)
Comment (atlldwp) [reply views/administration/school.hbs]: here updated the name to -->  school-administration-video-conference-checkbox

File: views/administration/school.hbs (hunk 3 above)
Comment (atlldwp) [reply views/administration/school.hbs]: here updated the name to -> 'school-administration-save-general-setting'

File: views/calendar/card.hbs [hunk 4]
```diff
@@ -43,7 +43,7 @@
               <span class="muted video-conference" style="display: none;">{{$t "calendar.text.videoConferenceDetailsAreLoading" }}</span>
               <!-- state: RUNNING -->
               <i class="fa fa-video-camera video-conference join-conference" aria-hidden="true" style="display: none;">&nbsp;</i>
-              <a class="video-conference join-conference" href="/videoconference" style="display: none;">{{$t "calendar.text.participateInVideoConference" }}</a>
+              <a class="video-conference join-conference" href="/videoconference" style="display: none;" data-testid="participate-video-conf-link">{{$t "calendar.text.participateInVideoConference" }}</a>
```
Comment (MartinSchuhmacher) [inline views/calendar/card.hbs:46]: Just to be consistrnt, please use the full "conference" instead of only "conf"

**Review by MartinSchuhmacher — CHANGES_REQUESTED** [review]

File: views/calendar/card.hbs (hunk 4 above)
Comment (atlldwp) [reply views/calendar/card.hbs]: yes, updated it to 'conference'.

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**